rever.xsh

```diff
@@ -0,0 +1,47 @@
+from rever.activity import activity
+from rever.conda import run_in_conda_env
+
+$PROJECT = 'versioned-hdf5'
+
+@activity
+def run_tests():
+    # Don't use the built-in pytest action because that uses Docker, which is
+    # overkill and requires installing Docker
+    with run_in_conda_env(['python=3.8', 'pytest', 'numpy', 'h5py',
+                           'ndindex', 'pyflakes', 'pytest-cov']):
```

Comment:
This looks like it is missing some dependencies compared to what is installed in Travis.